lib/tests/modules/gvariant.nix

```diff
@@ -1,44 +1,36 @@
 { config, lib, ... }:
 
-let inherit (lib) concatStringsSep mapAttrsToList mkMerge mkOption types gvariant;
-in {
-  options.examples = mkOption { type = types.attrsOf gvariant; };
+{
+  options = {
+    examples = lib.mkOption { type = lib.types.attrs; };
+    assertion = lib.mkOption { type = lib.types.bool; };
+  };
 
   config = {
-    examples = with gvariant;
-      mkMerge [
-        { bool = true; }
+    examples = with lib.gvariant;
+      lib.mkMerge [
```

Comment:
Since we are no longer testing the merging functionality, we can just drop the `mkMerge` and use a single big attribute.

Reply:
I removed it. Maybe we can add some tests for mkMerge in the future.